growus/growus.py

```diff
@@ -46,5 +51,43 @@ def api_workout_random():
 	response = Response(js, status=200, mimetype="application/json")
 	return response
 
+# inserts a single exercise for a user
+@app.route("/exercises/create", methods = ['POST'])
+def api_exercises_create():
+	exercise_type = request.form['type']
+	user_id = request.form['userId']
+	name = request.form['name']
+	description = request.form['description']
+
+	models.insert_exercise(exercise_type, user_id, name, description)
+	exercise = {"exercise_type" : exercise_type, 
+				"user_id" : user_id,
+				"name" : name,
+				"description" : description}
+
+	js = json.dumps(exercise)
+	response = Response(js, status=200, mimetype="application/json")
+	return response
+
+# remove all exercises associated with a user
+@app.route("/exercises/clear", methods = ['POST'])
+def api_exercises_clear():
+	user_id = request.form['userId']
```

Comment:
Can you change it to expect a json response rather than form input? also the other endpoints (: